Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM node:lts-buster
```

Comment:
What's the size of this image?

I'd take a look at https://github.com/libp2p/js-libp2p-webrtc-star/pull/223 to see the improvements made there. Buster likely has a lot more than is needed.

Reply:
Very smaller this one! thanks